### PR TITLE
pacman-query: add page

### PR DESCRIPTION
**Pull request by navarroaxel**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

For: #5256

### CODE REVIEW

File: pages/linux/pacman-query.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- Display information about an installed package:
+
+`pacman --query --info {{package_name}}`
+
+- List files owned by a package:
+
+`pacman --query --list {{package_name}}`
+
+- List orphan packages (installed as dependency not required by any):
```
Comment (bl-ue) [inline pages/linux/pacman-query.md:26]:
```suggestion
- List orphan packages (installed as dependencies but not required by any package):
```

File: pages/linux/pacman-query.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+- List installed packages and versions:
+
+`pacman --query`
+
+- List only the explicitly installed packages and versions:
+
+`pacman --query --explicit`
+
+- Find which package owns a certain file:
```
Comment (bl-ue) [inline pages/linux/pacman-query.md:14]:
```suggestion
- Find which package owns a file:
```

File: pages/linux/pacman-query.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
+# pacman --query
+
+> Arch Linux package manager utility.
+> More information: <https://man.archlinux.org/man/pacman.8>.
+
+- List installed packages and versions:
+
+`pacman --query`
+
+- List only the explicitly installed packages and versions:
```
Comment (bl-ue) [inline pages/linux/pacman-query.md:10]:
```suggestion
- List only packages and versions that were explicitly installed:
```

**Review by bl-ue — CHANGES_REQUESTED** [review]

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @navarroaxel!

**Pull request merged**